db/migrate/20230713050401_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :Text
```

Comment:
- [ ] Kindly follow Ruby convention for data attributes, `Title` should be `title`.

Reply:
Attributes updated